go/channelling/server/config.go

```diff
@@ -147,6 +147,7 @@ func NewConfig(container phoenix.Container, tokens bool) (*channelling.Config, e
 		RoomTypeDefault:                 defaultRoomType,
 		RoomTypes:                       roomTypes,
 		RoomNameCaseSensitive:           container.GetBoolDefault("app", "caseSensitiveRooms", false),
+		LockedRoomJoinableWithPIN:       container.GetBoolDefault("app", "lockedRoomJoinableWithPIN", true),
```

Comment:
This doesn't match the comment in `server.conf.in` which states that the default is `false`.